src/bitcoin-cli.cpp

```diff
@@ -595,7 +595,7 @@ class NetinfoRequestHandler : public BaseRequestHandler
         // Report peer connection totals by type.
         result += "        ipv4    ipv6   onion   total  block-relay\n";
         const std::array rows{"in", "out", "total"};
-        for (uint8_t i = 0; i < m_networks.size(); ++i) {
+        for (uint8_t i = 0; i < 3; ++i) {
```

Comment:
why not `rows.size()` then? 
even 23324 is in this PR let's make a proper fix without wiggling there-n-back

Reply:
To make the diff match upstream. Without the middle commit, the diff goes from `m_networks.size()` to `rows.size()`.